dev/TeachingTip/TestUI/TeachingTipPage.xaml.cs

```diff
@@ -868,5 +878,22 @@ private FrameworkElement getTeachingTipRoot(UIElement cancelClosesCheckBox)
             }
             return (FrameworkElement)current;
         }
+
+        public bool IsPageRTL
+        {
+            get
+            {
+                return FlowDirection == FlowDirection.RightToLeft;
+            }
```

Comment:
If using a new enough C#, there's a nice new syntax for this:

get => this.FlowDirection == FlowDirection.RightToLeft

Reply:
I did it but I'm not sure I find it more readable next to the old style set syntax.